airbyte-cdk/python/airbyte_cdk/connector_builder/message_grouper.py

```diff
@@ -83,8 +83,10 @@ def get_message_groups(
                     latest_config_update = message_group
             elif isinstance(message_group, AuxiliaryRequest):
                 auxiliary_requests.append(message_group)
-            else:
+            elif isinstance(message_group, StreamReadSlices):
                 slices.append(message_group)
+            else:
```

Comment:
mostly to make mypy happy, but this is a reasonable check

Reply:
This is very important. In Java, you usually use switch/case and the compiler can emit a warning if some cases aren't managed based on the typing. I like it personally